packages/koenig-lexical/src/plugins/DragDropPastePlugin.jsx

```diff
@@ -1,40 +1,41 @@
 import React from 'react';
-import {DRAG_DROP_PASTE} from '@lexical/rich-text';
 import {useLexicalComposerContext} from '@lexical/react/LexicalComposerContext';
 import {COMMAND_PRIORITY_LOW} from 'lexical';
 import {getEditorCardNodes} from '../utils/getEditorCardNodes';
 import {INSERT_IMAGE_COMMAND} from '../nodes/ImageNode';
 import {INSERT_AUDIO_COMMAND} from '../nodes/AudioNode';
```

Comment:
It would be great if we can move this onto the node itself similar to what we do for menu items, that way the drop/paste behaviour is generic and will work with whichever nodes are currently loaded in the editor.